weave

```diff
@@ -2010,17 +2010,10 @@ EOF
     launch-proxy)
         deprecation_warnings "$@"
         check_not_running proxy $PROXY_CONTAINER_NAME $BASE_EXEC_IMAGE
+        check_not_running_v2_plugin
         launch_proxy "$@"
         echo $PROXY_CONTAINER
         ;;
-    launch-plugin)
-        if ! check_running $CONTAINER_NAME 2>/dev/null ; then
-            echo "ERROR:" $CONTAINER_NAME "container must be running before plugin can be launched" >&2
-            exit 1
-        fi
-        launch_plugin_if_not_running "$@"
-        echo $PLUGIN_CONTAINER
-        ;;
     env|proxy-env)
```

Comment:
Are we going to print some message in a case of `weave run`? If yes, then I should probably print some warning as well.